certified-connectors/Cognizant Automation Center/apiProperties.json

```diff
@@ -0,0 +1,76 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "cacUrl": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "tabIndex": 1,
+            "clearText": true,
+            "required": "true"
+          },
+          "description": "Automation Center URL",
+          "displayName": "Automation Center URL",
+          "tooltip": "Provide Your AutomationCenter url"
+        }
+      },
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "Automationcenter API Key",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",

```

Comment:
@SelvaGanesh-Cognizant You have agreed not to have this iconBrandColor value for your connector.

Reply:
Hi @SelvaGanesh-Cognizant, can you please change the iconBrandColor? Thank you very much.